cmake/load_bundled_libraries.cmake

```
@@ -30,7 +30,7 @@ if(${vt_external_fmt})
   else()
     message(STATUS "vt_external_fmt = ON but neither fmt_DIR nor fmt_ROOT is provided!")
   endif()
-  find_package(fmt 7.1.0 REQUIRED)
+  find_package(fmt 11.1.3 REQUIRED)
```

```suggestion
  find_package(fmt 10.2.1 REQUIRED)
```

We should consider using a lower requirement. For example `distBVH` uses `10.2.1` ([see CI setup](https://github.com/sandialabs/distBVH/blob/9a296b5cf7886d75da051b39c5705bb6c0e6c1cc/spack.yaml#L8)), which builds and runs fine with the latest `develop`.